### PR TITLE
adding README and minor script updates for generating XCFramework

### PR DESCRIPTION
**Pull request by heckj**
Thank you for the help in slack getting to know how this works. While working through this, I made a few minor modifications to the script, and noted how the process works in the README, for anyone else wandering through and wanting to do the same.

### CODE REVIEW

File: README.md
Comment (lightsprint09) [inline README.md]: Automerge. Mo camel case

File: README.md
Comment (lightsprint09) [inline README.md]: Is it uppercase rust?

**Review by lightsprint09 — APPROVED** [review]
This looks awesome. When reading it, It sound quite clear. The script improvement help to streamline the process as well.

**Comment by lightsprint09** [comment]
Is this ready to merge?

**Comment by heckj** [comment]
Yep, I think so - wasn't sure of the etiquette for this repository/project

**Comment by lightsprint09** [comment]
Perfect. There is no etiquette. Is merging by yourself enabled once I approve it? If so go ahead and merge it.

**Comment by heckj** [comment]
Ah - I don't have permissions on your repository to merge You'd need to make me a collaborator (give me "write" permissions), or I need to defer to you or someone else that has write (or better) permissions to merge it.

**Comment by lightsprint09** [comment]
I would suggest I merge this for now and we move this repo into the automerge organisation soon. Once done you just need access to the organisation to contribute to automerge-swift and this repo.

**Comment by heckj** [comment]
Totally works for me

**Pull request merged**
